relationship_server.py

```diff
@@ -11,6 +11,41 @@
 nlp = spacy.load("en_core_web_lg")
 
 logger = logging.getLogger(__name__)
+
+
+'''
+    Utliity method to convert keys and values in a dictionary to lowercase.
+'''
+def makeDictLowercase(d):
```

Comment:
Please follow PEP8 conventions on function naming e.g. [snake_case over mixedCase](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

This code base has followed that for variables and function names, so cleaner not to mix conventions. If I missed it in come place, please mark that.

Reply:
Done via 4164190571431e1a98599a07c209883d1b8c0eeb